examples/subplots_axes_and_figures/subplots_adjust.py

```diff
@@ -3,6 +3,7 @@
 Subplots Adjust
 ===============
 
+Example showing the adjusting of spacing and margins of subplots.
```

Comment:
Adjusting the spacing of margins and subplots using :func:`~matplotlib.pyplot.subplots_adjust`: